Gemfile

```
@@ -73,14 +74,13 @@ gem "ruby-progressbar",                 "~>1.7.0",           :require => false
 gem "rubyzip",                          "~>2.0.0",           :require => false
 gem "rugged",                           "~>1.1",             :require => false
 gem "snmp",                             "~>1.2.0",           :require => false
-gem "sprockets",                        "~>3.7.2",           :require => false
-# stringio is an indirect dependency (part of rails internal gems)
-# CVE-2024-27280
-# resolved in ruby 3.0.7, ruby 3.1.5, and ruby >=3.2
+gem "sprockets",                        "~>3.7.2",           :require => false # indirect dependency CVE-2024-27280 (resolved in ruby 3.0.7, ruby 3.1.5, and ruby >=3.2)
 gem "stringio",                         "~>3.0.3",           :require => false
 gem "sync",                             "~>0.5",             :require => false
 gem "sys-filesystem",                   "~>1.4.5"
 gem "terminal",                                              :require => false
+gem "time",                             "~> 0.1.1"                             # CVE-2023-28756 fixed: ruby 3.0.6 - https://github.com/advisories/GHSA-fg7x-g82r-94qc
+gem "uri",                              "~> 0.10.0.3"                          # CVE-2023-28755 CVE-2023-36617 https://github.com/advisories/GHSA-hww2-5g85-429m
```

Are we locking down to 0.10.0?  Should this instead be

```suggestion
gem "uri",                              "~> 0.10.0", ">= 0.10.0.3"             # CVE-2023-28755 CVE-2023-36617 https://github.com/advisories/GHSA-hww2-5g85-429m
```